docs/Contributing.asciidoc

```diff
@@ -431,10 +489,245 @@ Also find more details in
+
+### distrobox container ###
+
+# Leap 15.6
+zypper addrepo -G -p 95 https://download.opensuse.org/repositories/devel:/openQA/15.6 devel-openqa \
+    && zypper addrepo -G -p 95 https://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.6/15.6/ devel-openqa-leap
+
+# Tumbleweed
+#zypper addrepo -G p 95 https://download.opensuse.org/repositories/devel:/openQA/openSUSE_Tumbleweed devel-openqa \
+#        && zypper addrepo -G -p 95 https://download.opensuse.org/repositories/SUSE:/CA/openSUSE_Tumbleweed/SUSE:CA.repo
+
+zypper -n ref && zypper -n install ca-certificates-suse \
+        os-autoinst-devel openQA-devel \
+        os-autoinst-scripts-deps \
+        os-autoinst-scripts-deps-devel \
+        crudini
+# plus your favourite dev tools, e.g. tmux
+----
+Note: https://github.com/pixelb/crudini[`crudini`] is a tool for manipulating
+values in ini files.
+
+Time to grab a hot beverage while installing…
+
+==== Git
+
+[source,sh]
+----
+### distrobox container ###
+# Remember to source this file
+source ~/localopenqa.leap.sh
+mkdir $OPENQA_BASEDIR
+mkdir $OPENQA_REPOS
+mkdir $OPENQA_BASEDIR/openqa $OPENQA_CONFIG $OPENQA_BASEDIR/perl5 $OPENQA_BASEDIR/openqa/db
+
+git clone git@github.com:os-autoinst/openQA $OPENQA_REPOS/openQA
+git clone git@github.com:os-autoinst/os-autoinst $OPENQA_REPOS/os-autoinst
+git clone git@github.com:os-autoinst/scripts $OPENQA_REPOS/scripts
+----
+
+==== Config Files
+
+[source,sh]
+----
+### distrobox container ###
+echo <<< EOM
+[localhost]
+key = 1234567890ABCDEF
+secret = 1234567890ABCDEF
+EOM >> $OPENQA_CONFIG/client.conf
+
+echo <<< EOM
+[production]
+dsn = dbi:Pg:dbname=openqa-local;host=127.0.0.1;port=5432
+user = geekotest
+password = x
+EOM >> $OPENQA_CONFIG/database.ini
+
+cp $OPENQA_REPOS/openQA/etc/openqa/workers.ini $OPENQA_CONFIG/workers.ini
+crudini --set $OPENQA_CONFIG/workers.ini global HOST http://localhost:9526
+crudini --set $OPENQA_CONFIG/workers.ini global WORKER_HOSTNAME 127.0.0.1
+
+cp $OPENQA_REPOS/openQA/etc/openqa/openqa.ini $OPENQA_CONFIG/openqa.ini
+crudini --set $OPENQA_CONFIG/openqa.ini auth method Fake
+----
+
+
+==== Build openQA
+
+[source,sh]
+----
+### distrobox container ###
+# Build steps
+cd $OPENQA_REPOS/os-autoinst
+make
+cd $OPENQA_REPOS/openQA
+make node_modules
+
+# Optionally check if you can access the database
+#psql -U geekotest -h 127.0.0.1 -p 5432 openqa-local
+
+# Initialize the database schema. This is using the database.ini you
+# created above.
+$OPENQA_REPOS/openQA/script/initdb --init_database
+----
+
+==== Run openQA
+
+Ready!
+
+For playing with the web UI, you just need to run this one daemon.
+For running openQA jobs, you need all.
+
+[source,sh]
+----
+### distrobox container ###
+# Main web UI (required for web interface and web API)
+$OPENQA_REPOS/openQA/script/openqa-webui-daemon
+
+# Websockets (for workers to connect so they can receive job assignments)
+openqa-websockets daemon
```

Comment:
Maybe add `$OPENQA_REPOS/openQA/script/` to the other services as well. All those hae to run from the script/ dir

Reply:
Actually I removed the path from this one.
Because` $OPENQA_REPOS/openQA/script` is already in `$PATH` if you source the development script.